docs/source/usage/workflows/memoryPerDevice.py

```diff
@@ -27,8 +27,9 @@
 
 cell_size = 0.8e-6 / 384.  # 2.083e-9 m
 
-y0 = 0.5e-6  # position of foil surface (m)
-y1 = 1.0e-6  # target thickness (m)
+y0 = 0.5e-6  # position of foil front surface (m)
+delta_y = 1.0e-6  # target thickness (m)
+y1 = y0 + delta_y # position of the foil rear surface (m)
```

Comment:
Usage of `the` has now become a little inconsistent between the comments on the front and rear surfaces.

Reply:
in this line there needs to be one additional space before the `#` to agree with the pep8 standard

Reply:
Yep, double space between last character and comment in Python